README.md

```diff
@@ -15,6 +15,11 @@ Download and build Linux kernel from scratch:
 make build/bzImage
 ```
 
+Download and build Busybox for root file system from scratch:
+```shell
+make build/rootfs.cpio
```

Comment:
Let's create a new target `rootfs` for short. Therefore, we can simply run `make rootfs`.